.github/workflows/publish_client_android_release_candidate.yml

```diff
@@ -0,0 +1,68 @@
+name: Publish Client Android Release Candidate
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: "SHA of commit to publish (example: 2ac8bd4a915a7c8a3e3a63081d72da33baa9ea7e)"
+        required: true
+
+jobs:
+  release:
+    name: Publish Client Android Release Candidate
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    environment: Releases Repository
+    env:
+      RELEASES_REPOSITORY: ${{ secrets.RELEASES_REPOSITORY }}
+      RELEASES_DEPLOY_KEY: ${{ secrets.RELEASES_DEPLOY_KEY }}
+      RELEASE_COMMIT_SHA: ${{ github.event.inputs.commit_sha }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Download Release Candidate
+        uses: dawidd6/action-download-artifact@575b1e4167df67acf7e692af784566618b23c71e
+        with:
+          workflow: build_release_candidates.yml
+          commit: ${{ env.RELEASE_COMMIT_SHA }}
+          name: client_android_release_${{ env.RELEASE_COMMIT_SHA }}
+          path: client_android_release
+
+      - name: Checkout Releases Repository
+        uses: actions/checkout@v2.3.4
+        with:
+          repository: ${{ env.RELEASES_REPOSITORY }}
+          ssh-key: ${{ env.RELEASES_DEPLOY_KEY }}
+          path: outline-releases
+
+      - name: Push Tags and Release Commits
+        run: |
+          git config --global user.name "OutlineBot"
+          git config --global user.email "outlinebot@users.noreply.github.com"
+
+          RELEASE_VERSION="$(node scripts/get_version.mjs android)"
+          RELEASE_TAG="android-${RELEASE_VERSION}"
+
+          [ "$(git tag -l "$RELEASE_TAG")" != "" ] && exit 1
```

Comment:
Okay, I ran into a race condition when trying to publish multiple releases at once. This pre-checks the tag so we don't get into the situation where the push to master fails (due to the race condition) but we still tag the commit, forcing us to create a new PR with a new version. Now if one of our publishes fails, we can just re-run it and it's fine, it's good enough.